src/components/molecules/SubtabGroup/SubtabGroup.tsx

```diff
@@ -0,0 +1,32 @@
+import { RadioGroup } from '@headlessui/react';
+import { SubTab } from 'src/components/atoms';
+
+export const SubtabGroup = ({
+    options,
+    selectedOption,
+    handleClick,
+}: {
+    options: string[];
+    selectedOption: string;
+    handleClick: (option: string) => void;
+}) => {
+    return (
+        <RadioGroup
+            value={selectedOption}
+            onChange={handleClick}
+            as='div'
+            className='flex flex-row items-center gap-1 rounded-lg bg-tabBg px-1.5 py-[5px] laptop:gap-2 laptop:p-2'
```

Comment:
the background might be neutral-800

Reply:
@as472 Yeah that is the case in the design system file, but in the hyperliquid ticket design it's updated: https://www.figma.com/design/o2wjskNCdkt65haAgWPxo6/Product-UI-(Main)?node-id=15836-33212&m=dev

Will inform @benk-sf to update it in design system

Reply:
okay. sure